File: SalesTaxCalculationKata.Data/Models/Order.cs
﻿using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace SalesTaxCalculationKata.Data.Models
{
    [Table("Orders")]
    public class Order
    {
        public int OrderId { get; set; }
        public virtual ICollection<OrderItem> OrderItems { get; set; }
        public decimal SalesTaxTotal { get; set; }
        public decimal GrandTotal { get; set; }
        public bool IsComplete { get; set; }
    }
}

File: SalesTaxCalculationKata.Web/src/order.ts
import { bindable } from "aurelia-framework";
import { OrderItemModel } from "./models/order-model";

export class Order {
  @bindable orderId: number;
  @bindable orderItems: Array<OrderItemModel>;
  @bindable salesTaxTotal: number;
  @bindable grandTotal: number;
  @bindable isComplete: boolean;
}
